app/assets/stylesheets/static_pages.scss

```diff
@@ -4,45 +4,49 @@
 
 @import "ashby_settings.scss";
 
-.banner_text {
-  font-style: bold;
-  padding-bottom: 150px;
```

Comment:
I don't remember if I added this in as a quick fix before that one GM, but this isn't sustainable lol. If we use the grid, we can just make sure this text is in the same `row`